Neos.Neos/Resources/Private/Fusion/RawContentMode.fusion

```diff
@@ -13,9 +13,44 @@ prototype(Neos.Neos:RawContent) < prototype(Neos.Fusion:Template) {
 			columnContent = Neos.Neos:ContentCollection {
 				nodePath = '.'
 			}
-
 		}
 	}
+
+  prototype(Neos.Neos:ContentCase) {
+    default {
+      condition = Neos.Fusion:CanRender {
+        type = ${q(node).property('_nodeType.name')}
+      }
+    }
+    fallback {
+      @position = 'after default'
+      condition = true
+      type = 'Neos.Neos:RawContentFallbackRenderer'
+    }
+  }
+}
+
+prototype(Neos.Neos:RawContentFallbackRenderer) < prototype(Neos.Neos:ContentComponent) {
```

Comment:
should this get an own prototype file? or rather not since its not api ...

Reply:
I didn't want to put too much effort into this, as Martin anyway replaces everything in 8.1 🤷🏻‍♂️